content/en/edge-computing.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Edge Computing
+status: Feedback Appreciated
+category: Technology
+---
+
+## What it is
+
+Edge computing is a [distributed system](/distributed-systems/) approach that shifts some storage and computing capacity from the primary data center to the data source.
+The gathered data is computed locally (e.g., on a factory floor, in a store, or throughout a city) rather than sent to a centralized data center for processing and analysis. 
+These local processing units or devices represent the system's edge, whereas the data center is its center.
+The output computed at the edge is then sent back to the primary data center for further processing.
+Examples of edge computing include gadgets on our wrists or computers that analyze traffic flow.
+
+## Problem it addresses
+
+Due to the enormous amounts of data that Internet of Things (IOT)devices create, data has become the next important asset for enterprises worldwide. 
+For critical business strategies,firms also increasingly need to examine the data as fast as possible, But Organisations are adjusting for the latency and performance issues.
+Previously, businesses would transfer all their monitoring data to a corporate data center or the cloud for processing, analysis, and storage. However, as the IoT has expanded, the amount of data available makes this method impractical.
+So, moving the computing infrastructure physically closer or next to the edge where the data creates will solve the problem of performance and latency. 
```

Comment:
```suggestion

```
I don't think edge computing is about moving the infrastructure closer but about having some of the processing and analysis (not all!) happening on the device before being sent to the datacenter. The datacenter is where it used to be. Also, this is discussion is moving into the how it help.